WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -66,15 +67,39 @@
             <% else %>
               <%= alert :success, t('registrations.accepted') %>
             <% end %>
+            <div>
+              <% if @competition.connected_stripe_account_id %>
+                <% paid_entry_fees = humanized_money_with_symbol @registration.paid_entry_fees %>
+                <% outstanding_entry_fees = humanized_money_with_symbol @registration.outstanding_entry_fees %>
+                <% entry_fee = humanized_money_with_symbol @registration.entry_fee %>
+                <% if @registration.outstanding_entry_fees > 0 %>
+                  <% if @registration.paid_entry_fees > 0 %>
+                    <%= alert :info, t('registrations.entry_fees_partially_paid', paid: paid_entry_fees, total_entry_fee: entry_fee) %>
+                  <% end %>
+                  <%= form_tag competition_process_payment_path do %>
+                    <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                      data-key="<%= ENVied.STRIPE_PUBLISHABLE_KEY %>"
+                      data-name="World Cube Association"
+                      data-description="Registration fee"
+                      data-amount="<%= @registration.outstanding_entry_fees.cents %>"
+                      data-currency="<%= @registration.entry_fee_currency_code %>"
+                      data-email="<%= @registration.user.email %>"
+                      data-zip-code="false"
+                      data-label="<%= t('registrations.payment_button_text', entry_fee: outstanding_entry_fees) %>"
+                      data-locale="auto">
```

Comment:
Could you work with @viroulep to figure out what this should be set to? Maybe all we need to do here is make an issue for this for @viroulep to deal with later.